release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2857,6 +2857,40 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.8
+[id="ocp-4-17-8_{context}"]
+=== RHSA-2024:10818 - {product-title} {product-version}.8 bug fix, and security update advisory
+
+Issued: 11 December 2024
+
+{product-title} release {product-version}.8 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:10818[RHSA-2024:10818] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:10821[RHBA-2024:10821] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.8 --pullspecs
+----
+
+[id="ocp-4-17-8-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the provider ID for IBMPowerVSMachine's object did not populate properly due to an improper retrieval of the IBM Cloud Workspace ID. As a result, certificate signing requests (CSRs) were pending in the hosted cluster. With this release, the provider ID successfully populates and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44880[*OCPBUGS-44880*])
+
+* Previously, you could not remove a finally pipeline task from the edit Pipeline form if you created a pipeline with only one finally task.  With this change, you can remove the finally task from the edit Pipeline form and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44873[*OCPBUGS-44873*])
+
+* Previously, if you used the `oc adm node-image create` command and the image generation step failed, it reported a simple error and did not show the container's log. As a result, the simple error does not show the underlying issue that caused the image generation step to fail. With this release, the `oc adm node-image create` command shows the container's log. (link:https://issues.redhat.com/browse/OCPBUGS-44508[*OCPBUGS-44508*])
+
+* Previously, if you created load balancers for a cluster that you wanted to install on {ibm-power-name} and the creation of the load balancers timed out, the installation of the cluster failed and did not report the error. The cluster failed because both internal and external DNS load balancer names were not created. With this release, if internal and external DNS load balancer names do not exist during cluster installation, the installation program outputs an error so that you have an opportunity to add the names so that the cluster installation process continues. (link:https://issues.redhat.com/browse/OCPBUGS-44247[*OCPBUGS-44247*])
```

Comment:
```suggestion
* Previously, if you created load balancers for a cluster that you wanted to install on {ibm-power-name} and the creation of the load balancers timed out, the installation of the cluster failed and did not report the error. The cluster failed because both internal and external DNS load balancer names were not created. With this release, if internal and external DNS load balancer names do not exist during cluster installation, the installation program produces an error notification, and you can add the names so that the cluster installation process continues. (link:https://issues.redhat.com/browse/OCPBUGS-44247[*OCPBUGS-44247*])
```
Revision to avoid using "output" as a verb, per the ISG.